es/resource_elasticsearch_kibana_alert.go

```diff
@@ -152,7 +153,7 @@ func resourceElasticsearchKibanaAlert() *schema.Resource {
 				DiffSuppressFunc: suppressEquivalentJson,
 			},
 			"actions": {
-				Type:        schema.TypeSet,
+				Type:        schema.TypeList,
```

Comment:
Why switch this from a set to a list?

Reply:
Without the change, I got a state change every time: https://github.com/phillbaker/terraform-provider-elasticsearch/pull/335#issuecomment-1424059053